apps/src/lab2/types.ts

```diff
@@ -38,7 +49,7 @@ export interface ProjectSources {
 }
 
 // We will eventually make this a union type to include other source types.
-export type Source = BlocklySource;
+export type Source = BlocklySource | MultiFileSource;
```

Comment:
Do you mind explaining the difference between `Source` type here which includes `MultiFileSource` and is included as a property in `ProjectLevelData` and `source` in `LevelProperties` which has type `MultiFileSource`?

Reply:
This is super helpful. And both your cleanup items SGTM, especially the first bullet. Unsure how involved the process of converting stored legacy Blockly from stringified to un-stringified JSON.
Thanks so much!